src/viewport.js

```diff
@@ -805,21 +805,24 @@ $.Viewport.prototype = {
      * @return {OpenSeadragon.Viewport} Chainable.
      * @fires OpenSeadragon.Viewer.event:zoom
      */
-    zoomTo: function( zoom, refPoint, immediately ) {
+    zoomTo: function(zoom, refPoint, immediately) {
 
         this.zoomPoint = refPoint instanceof $.Point &&
             !isNaN(refPoint.x) &&
             !isNaN(refPoint.y) ?
             refPoint :
             null;
 
-        if ( immediately ) {
-            this.zoomSpring.resetTo( zoom );
+        if (immediately) {
+            var _this = this;
```

Comment:
I think it's a good practice to put the `var _this` at the top of the function (unlike other variables where I prefer to keep them near where they're used) since in the future it might be needed elsewhere in the function.